rustls/src/versions.rs

```diff
@@ -65,8 +67,18 @@ pub static TLS13_VERSION: &Tls13Version = &Tls13Version {};
 ///
 /// There is one value of this type.  It is `TLS12_VERSION`.
 #[non_exhaustive]
-#[derive(Debug, Eq, PartialEq)]
-pub struct Tls12Version {}
+#[derive(Debug)]
+pub struct Tls12Version {
+    pub(crate) client: &'static dyn crate::client::Tls12Handler,
+}
+
+// TODO
```

Comment:
When/why?

Reply:
impl'd this for `SupportedProtocolVersion` instead